lib/lit/i18n_backend.rb

```diff
@@ -62,21 +63,44 @@ def lookup(locale, key, scope = [], options = {})
         new_content = @cache.init_key_with_value(key_with_locale, content)
         content = new_content if content.nil? # Content can change when Lit.humanize is true for example
 
-        if content.nil? && options[:default].present?
-          if options[:default].is_a?(Array)
-            default = options[:default].map do |key|
-              if key.is_a?(Symbol)
-                I18n.normalize_keys(nil, key.to_s, options[:scope], options[:separator]).join('.').to_sym
-              else
-                key
+
+        # so there is no content in cache - it might not be if ie. we're doing
```

Comment:
Those changes are from another PR - https://github.com/prograils/lit/pull/83/commits. Strange. Maybe you should rebase from master first?